properties/properties/devenv-dataitemtableview-property.md

```diff
@@ -20,7 +20,7 @@ Data items on reports.
 
 ## Examples
 
-In the following example, the **DataItemView** property is used to sort a table view based on the `"Entry No."` field.
+In the following example, the **DataItemTableView** property is used to sort a table view based on the `"Entry No."` field.
```

Comment:
How did you find this file? It appears to me to not being used at all. 

Shouldn't the file be deleted, @SusanneWindfeldPedersen ?

Reply:
After is saw it on MSFT learn, i just searched for in the repository and 2 hits came out...
I decided to comment for both